tests/test_2d_segmentation.py

```diff
@@ -47,7 +48,9 @@ def testOutputFiles(self):
         # Test if the amount of binary buffers is equals to the elements in `parts`
         output_files = os.listdir(os.path.join(self.inference_test_dir, self.output_dir))
         count_masks = len([f for f in output_files if f.startswith("output_masks_")])
-        self.assertEqual(count_masks, len(data['parts']))
+        segmentation_masks_parts = [part for part in data['parts'] if part['binary_type']
+               not in DICOM_BINARY_TYPES]
+        self.assertEqual(count_masks, len(segmentation_masks_parts))
 
         for index, part in enumerate(data['parts']):
```

Comment:
```suggestion
        for index, part in enumerate(segmentation_masks_parts):
```
This should be updated as well as I understand?

Reply:
yes, I forgot ^^